BetaCameras/ifm/O3D3xx.cs

```diff
@@ -207,19 +283,15 @@ public O3D3xx(int applicationId = -1)
             CameraIP = "192.168.1.172";
```

Comment:
What's this hard-coded IP doing here?

Reply:
I would assume it's there as a default setting.
But I agree, it is quite confusing. Actually this was the issue I was having yesterday trying to connect to the camera. I set the `SerialNumber` instead of the `CameraIP`. It took me quite a while to figure out that the IP the camera was trying to connect to was 172 instead of 232.
I think it would be better to not be able to connect at all as long as no specific IP has been set.

Reply:
Yes. Then remove the hard-coded one and throw a meaningful exception if the IP isn't set when `Connect()` needs it.